atlp-rwanda/e-commerce-ones-and-zeroes-fn
# [Finishes #187355048] Configure storybook for component development

### Description
This pull request introduces the configuration for Storybook, a powerful tool for developing UI components. This setup will enhance our development workflow by allowing us to build, test, and review components independently of the main application.

### How to Test

1. Clone the Repository and Checkout the PR Branch:
`git checkout ch-storybook`
2. Install Dependencies: `npm install`
3. start storybook: `npm run storybook`
4. once started, we're going to see the Button used as example
![Screenshot 2024-06-18 200900](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/239ebab5-d2be-42b8-9e5c-3b18a5026ca0)


### Reference
[creating storybook stories]( https://storybook.js.org/docs/writing-stories/args)
